mrblib/mitamae/resource_executor/remote_directory.rb

```diff
@@ -59,7 +59,9 @@ def show_differences
         super
 
         if current.exist
-          diff = run_command(["diff", "-u", attributes.path, @temppath], error: false)
+          temppath = ::File.join('/tmp', Time.now.to_f.to_s)
+          run_command(['cp', '-r', ::File.join(@resource.recipe.dir, desired.source), temppath])
+          diff = run_command(["diff", "-u", attributes.path, temppath], error: false)
```

Comment:
FYI, the original intention of `@temppath` in Itamae *may* be supporting `itamae ssh`. But MItamae doesn't provide `itamae ssh`.